src/_common/shell/shell.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+	<app-scroll-inview-parent>
+		<app-theme />
+		<app-loading-bar />
+		<app-growls />
```

Comment:
Can't we pull app-modals here too?

Reply:
Not all sections used modals, so I didn't include it. We can totally put them if we want to be able to use them in all sections. It would add a little extra code to stuff like the gameserver, but it'd definitely be reasonable so we don't have to think about it.

Reply:
Added it to common shell.